examples/BERT/cross_lingual_mlm_task.py

```diff
@@ -197,6 +258,15 @@ def _forward(x):
                         help='path to load the reference model for evaluation')
     parser.add_argument('--mask_frac', type=float, default=0.15,
                         help='the fraction of masked tokens')
+    parser.add_argument('--dist', type=bool, default=False,
+                        help='run distributed version')
+    parser.add_argument('--world_size', type=int, default=3,
```

Comment:
Just a question to help me understand it. Why do we set the world size to 3 if we shard the model on two device?